ansible/roles/ansible-datamover-api/templates/trilio.list

```diff
@@ -1 +1 @@
-deb http://{{IP_ADDRESS}}:8085 deb-repo/
+deb [trusted=yes] https://apt.fury.io/triliovault/ /
```

Comment:
- As there are multiple occurrences of this repo address, lets add this to vars.yml file as a variable and use that